### DISCUSSION TITLE
How to display the red channel only

### DISCUSSION THREAD

**Discussion by icpz** [ImageMagick/ImageMagick#3778] [Help] [answered]
Hello!

I'm trying to view the red channel only with `magick display` command. I've tried `magick display -channel Red xxx.jpg`, but it doesn't seem working. The pop up window still shows my image with full colors.

What is the correct command to do that? Thanks!!

Here is my magick verison info:
```bash
$ magick --version
Version: ImageMagick 7.0.11-13 Q16 x86_64 2021-05-17 https://imagemagick.org
Copyright: (C) 1999-2021 ImageMagick Studio LLC
License: https://imagemagick.org/script/license.php
Features: Cipher DPC HDRI Modules OpenMP(4.0)
Delegates (built-in): bzlib fontconfig freetype gslib heic jng jp2 jpeg lcms lqr ltdl lzma openexr png ps tiff webp x xml zlib
```

**Comment by snibgo** [comment]
Try:
```
magick display -channel Red -separate +channel xxx.jpg
```

**Reply by icpz** [reply]
It gives me:

```display: unrecognized option \`-separate' @ error/display.c/DisplayImageCommand/1671.```

I didn't find `-separate` a valid option for `display`. Thanks for your reply!

**Comment by fmw42** [comment, marked as answer]
You might also want

`magick display -channel BG -threshold 100% xxx.jpg`

or

`magick display -channel BG -threshold 100% -colorspace gray xxx.jpg`

**Reply by icpz** [reply]
These commands work! Thank you very much!!

BTW I think the documentation of `display` could be more specific... ^_^
